### DISCUSSION TITLE
Email Sending Module along with Template builder (Add ESP as a module)

### DISCUSSION THREAD

**Discussion by nagireddybi** [postalserver/postal#2030] [Development discussions]
Hello Team

Is it possible to add the ESP module to postal, does it work?, for clients who want to send simple plain text emails, they can use it without the need of any third-party ESP modules. 

I am planning to add this module, please share your ideas and thoughts on the same. I really appreciate your time. 

Thanks
Nagi

**Comment by willpower232** [comment]
As per the docs https://docs.postalserver.io/welcome/faqs#can-you-add-mailing-list-feature

This isn't really what Postal was intended for but if you want to make the changes yourself then there is nothing really stopping you.
